Airport Ticket Booking System/Pricing/CurrencyConverterExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public static class CurrencyConverterExtensions
+{
+    private static readonly Dictionary<(Currency From, Currency To), decimal> ExchangeRates = new Dictionary<(Currency, Currency), decimal>
```

Comment:
`ExchangeRates` are not fixed, they change day by day. We would usually use a 3rd party service to retrieve those. This is just an FYI, no need to fix it here ;)